docs/CHANGELOG.md

```diff
@@ -13,7 +13,7 @@ This release is API compatible but contains a major internal change in the HTTP
     the fetcher at Updater initialization and explicitly depending on requests
 * ngclient: TLS certificate source was changed. Certificates now come from operating
   system certificate store instead of `certifi` (#2762)
-* Test infrastucture has improved and should now be more usable externally, e.g. in
```

Comment:
Is it OK to modify the CHANGLOG?

Reply:
in this part certainly: these are still unreleased changes. I think spelling fixes are fine in released ones as well